windows/sound.cpp

```diff
@@ -588,21 +580,190 @@ bool CSound::CheckSampleTypeSupported ( const ASIOSampleType SamType )
         ( SamType == ASIOSTInt32MSB24 ) );
 }
 
-bool CSound::CheckSampleTypeSupportedForCHMixing ( const ASIOSampleType SamType )
-{
-    // check for supported sample types for audio channel mixing (see bufferSwitch)
-    return ( ( SamType == ASIOSTInt16LSB ) ||
-             ( SamType == ASIOSTInt24LSB ) ||
-             ( SamType == ASIOSTInt32LSB ) );
-}
+struct sample16 {

```

Comment:
Wow, a lot of changes here. Have you written this code specifically for Jamulus or have you copied it from another (hopefully Open Source) project?

Reply:
This code was all written by me.